.github/testing/config.yml

```diff
@@ -0,0 +1,16 @@
+DRYDOCK_INIT_JOBS: true
+K8S_NAMESPACE: testing-openedx
+PLUGINS:
+- forum
+- mfe
+- minio
+- drydock
+DRYDOCK_FLOWER: true
+DRYDOCK_INGRESS: true
+DRYDOCK_DEBUG: true
+DRYDOCK_ENABLE_CELERY_TUNING: true
+DRYDOCK_ENABLE_MULTITENANCY: true
```

Comment:
There are a couple of features we should add here:
- NewRelic (through setting DRYDOCK_NEWRELIC_LICENSE_KEY)
- DRYDOCK_INGRESS_LMS_EXTRA_HOSTS aiming to add extra LMS tenants in the same [LMS ingress](https://github.com/eduNEXT/drydock/blob/v17.3.0/drydock/templates/drydock/k8s/ingress/lms.yml#L60) (by the way this variable is not documented in the [readme](https://github.com/eduNEXT/drydock?tab=readme-ov-file#configuration), we should add it)
- DRYDOCK_INGRESS_EXTRA_HOSTS aiming to add ingresses for additional IDAs like notes, and discovery, among others.
-  Pdb settings (there multiple variables controlling pdb for different deployments)